packages/suite/src/views/wallet/tokens/components/TokenList.tsx

```diff
@@ -108,123 +126,187 @@ export const TokenList = ({
-                            <StyledQuestionTooltip
-                                label="TR_TOKEN_UNRECOGNIZED_BY_TREZOR"
-                                tooltip="TR_TOKEN_UNRECOGNIZED_BY_TREZOR_TOOLTIP"
-                                $addMarginTop={!!knownTokens.length}
-                            />
+        knownTokens.length > 0 && (
+            <Table paddingType="none">
+                <Columns>
+                    <ColName>
+                        <Translation id="TR_TOKEN" />
+                    </ColName>
+                    <ColName>
+                        <Translation id="AMOUNT" />
+                    </ColName>
+                    {!isTestnet && (
+                        <>
+                            <ColName>
+                                <Translation id="TR_EXCHANGE_RATE" />
+                            </ColName>
+                            <ColName>
+                                <Translation id="TR_7D_CHANGE" />
+                            </ColName>
+                        </>
+                    )}
+                </Columns>
+                {knownTokens.map(t => (
+                    <Token key={t.contract}>
+                        <Cell>
+                            <TokenName>{blurUrls(t.name)}</TokenName>
+                        </Cell>
+                        <Cell $isBigger={isTestnet}>
+                            <Amount>
+                                {!isTestnet && (
+                                    <StyledFiatValue
+                                        amount={t.balance || '1'}
+                                        symbol={networkSymbol}
+                                        tokenAddress={t.contract as TokenAddress}
+                                        showLoadingSkeleton
+                                    />
+                                )}
+                                <CryptoAmount
+                                    value={t.balance}
+                                    customSymbol={
+                                        networkType === 'cardano'
+                                            ? undefined
+                                            : blurUrls(t.symbol?.toUpperCase())
+                                    }
+                                />
+                            </Amount>
+                        </Cell>
+                        {!isTestnet && (
+                            <>
+                                <Cell>
+                                    <StyledPriceTicker
+                                        symbol={networkSymbol}
+                                        contractAddress={t.contract as TokenAddress}
+                                    />
+                                </Cell>
+                                <Cell>
+                                    <TrendTicker
+                                        symbol={networkSymbol}
+                                        contract={t.contract as TokenAddress}
```

Comment:
Why different naming for contract address?

Reply:
9cbebb8f48e91767052ed845b741fa86d574c37c